pkg/config/config.go

```diff
@@ -0,0 +1,88 @@
+// config is the package containing configuration for fluxd, shared so
+// it can be used by fluxd itself as well as other programs e.g.,
+// `fluxctl install`.
+package config
+
+import (
+	"fmt"
+	"time"
+)
+
+const ConfigPath = "/etc/fluxd/conf"
+const ConfigName = "flux-config"
+const FluxConfigVersion = "v1"
```

Comment:
Can you put this in a single const block?

We should start doing some linting int e